PRIME-TU-Delft/interactive-visuals
# Set up a Docker image for our application

Add a Dockerfile that builds our application as a static page, and then serves it using nginx.

To test:
```bash
docker build app/ -t interactive-visuals
docker run -it --rm -p 80:80 interactive-visuals
```

(Binding to port 80 on your host machine requires root privileges or setting a capability. If you're running Docker in rootless mode, make sure it has permission to expose ports under 1024 using `sudo setcap CAP_NET_BIND_SERVICE=+eip /usr/bin/rootlesskit`)
